app-cdr/dvd+rw-tools/files/dvd+rw-tools-7.1-clang.patch

```diff
@@ -0,0 +1,71 @@
+--- a/dvd+rw-booktype.cpp	2006-06-24 12:21:48.000000000 +0300
++++ b/dvd+rw-booktype.cpp	2021-03-14 10:16:02.533741829 +0200
```

Comment:
Did you find this somewhere? We prefer to keep headers in patch files for traceability.

Reply:
No, I tried to compile it with clang and fixed the bugs.  There is a patch attached to one of the bugs, but it is not complete for clang 11.